StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -119,6 +130,7 @@ final class UpdatePaymentMethodViewController: UIViewController {
         }
         let label = ElementsUI.makeSmallFootnote(theme: viewModel.appearance.asElementsTheme)
         label.text = viewModel.footnote
+        label.isUserInteractionEnabled = false
```

Comment:
This should be in makeSmallFootnote(), the label should never be selectable, not just in this context.

Reply:
That's true, I didn't consider that— I'll fix that